docs/config.json

```diff
@@ -16,6 +16,10 @@
           "title": "Installation",
           "slug": "/installation/"
         },
+        {
+          "title": "End Users",
```

Comment:
I think this page should be an introductory page to `/docs/connect-your-client` rather than a page on its own. Other pages in "Home" don't really belong to a section (or could belong in multiple sections), but this one has a clear link to "Connect your Client". 

I also like the idea of having a single entry point to the "Connect your Client" section that puts each page in context/provides a summary of common commands, and I see this guide as serving that role.

Reply:
Moved / adjusted.